src/functions.rs

```diff
@@ -1772,6 +1772,96 @@ impl fmt::Display for FunctionArgument<'_> {
     }
 }
 
+// This seems to be a false positive on clippy's part
+// When I remove the return things don't compile correctly
```

Comment:
Are you sure you're removing the correct return?  The one on 1828 does look needless to me.